Cargo.toml

```diff
@@ -14,14 +14,14 @@ documentation = "https://docs.rs/prometheus"
 travis-ci = { repository = "pingcap/rust-prometheus" }
 
 [features]
-default = []
+default = ["protobuf"]
```

Comment:
seem we can make protobuf not default now. 
@breeswish

Reply:
Makes sense to me.